.mailmap

```diff
@@ -0,0 +1,18 @@
+Dongdong Tian <seisman.info@gmail.com>
+Dongdong Tian <seisman.info@gmail.com> <seisman@users.noreply.github.com>
+Jiayuan Yao <coreman.seism@gmail.com>
+Jiayuan Yao <coreman.seism@gmail.com> <core-man@users.noreply.github.com>
+Jing-Hui Tong <86273921+JingHuiTong@users.noreply.github.com> <r0922422020@g.ntu.edu.tw>
+Max Jones <meghanj@alum.mit.edu>
+Max Jones <meghanj@alum.mit.edu> <14077947+maxrjones@users.noreply.github.com>
+Max Jones <meghanj@alum.mit.edu> <meghanj@hawaii.edu>
+Max Jones <meghanj@alum.mit.edu> <meghanrjones@users.noreply.github.com>
+Michael Grund <michael_grund@gmx.de> <23025878+michaelgrund@users.noreply.github.com>
+Michael Grund <michael_grund@gmx.de> <michael.grund@kit.edu>
+Michael Grund <michael_grund@gmx.de> <michaelgrund@users.noreply.github.com>
+Wei Ji Leong <23487320+weiji14@users.noreply.github.com> <weiji.leong@vuw.ac.nz>
+Wei Ji Leong <23487320+weiji14@users.noreply.github.com> <weiji14@users.noreply.github.com>
+Will Schlitzer <schlitzer90@gmail.com> <willschlitzer@users.noreply.github.com>
+Yvonne Fröhlich <94163266+yvonnefroehlich@users.noreply.github.com> <yvonne.froehlich@kite.edu>
```

Comment:
Hm. Not sure where the email address ``yvonne.froehlich@kite.edu`` is coming from. The "e" after "kit" appears wrong to me. And then this entry is identical with the one in line 17.
```suggestion
```

Reply:
This email addresss comes from 0e630e3440b51ab6ac50d860c249276886cda07b. Maybe there was a typo in the commit message when I merged that PR.